java/ql/test/experimental/stubs/springframework-5.2.6/org/springframework/core/ParameterizedTypeReference.java

```diff
@@ -0,0 +1,27 @@
+package org.springframework.core;
+
+public abstract class ParameterizedTypeReference<T> {
+    public java.lang.reflect.Type getType() {
+        return null;
+    }
+
```

Comment:
There are already stubs in the repo for springframework-5.2.3; can we use those instead of adding these?

Reply:
fixed